avionics/common/include/states/ascent_to_apogee.h

```diff
@@ -55,20 +56,35 @@ class AscentToApogee : public IState {
     void onEntry() override {
         apogee_checks = 0;
         mach_checks = 0;
+        initial_time = 0;
         // Awkward to pass in first value - 0-initialize works with checks.
         last_alt = 0;
     }
 
+    StateId mach_timeout() {
```

Comment:
this should probably be a private method
Also, naming standard would use `machTimeout`